x/ccv/provider/keeper/proposal_test.go

```diff
@@ -917,6 +933,10 @@ func TestBeginBlockInit(t *testing.T) {
 
 	valAddr, _ := sdk.ValAddressFromBech32(validator.GetOperator())
 	mocks.MockStakingKeeper.EXPECT().GetLastValidatorPower(gomock.Any(), valAddr).Return(int64(1), nil).AnyTimes()
+
+	// for each validator, expect a call to GetValidatorByConsAddr with its consensus address
```

Comment:
Why "for each"? This seems to only be doing it for a _single_ validator?
nit: remove new line after comment